core/arch/arm/tee/tee_rpmb.c

```diff
@@ -305,20 +333,47 @@ static TEE_Result tee_rpmb_mac_calc(uint8_t *mac, uint32_t macsize,
 }
 
 struct tee_rpmb_mem {
-	paddr_t phpayload;
+	struct teesmc32_arg *arg;
+	paddr_t pharg;
 	paddr_t phreq;
+	paddr_t phreq_cookie;
 	paddr_t phresp;
+	paddr_t phresp_cookie;
 	size_t req_size;
 	size_t resp_size;
 };
 
+static void tee_rpmb_clear_session(struct tee_ta_session **session)
```

Comment:
These aren't needed any longer for RPC now that the user TA translation tables are assigned per thread instead of per CPU as it was in the past.